phoenix-ios/phoenix-ios/views/PaymentView.swift

```diff
@@ -370,12 +396,69 @@ fileprivate struct SummaryInfoGrid: InfoGridView {
 		}
 	}
 	
+	@ViewBuilder
+	var paymentMessaegRow: some View {
+		let identifier: String = #function
+		
+		// Note: We support AES too.
+		// But we don't have any real-world examples of its usage, and the spec doesn't given a clear example.
+		// What the heck do we get when we decode the ciphertext ?
+		// So, without an example, we're unsure of how to proceed.
```

Comment:
We should be able to decrypt it (with the stored preimage). And I **assume** it will decode to either `{"tag": "message", ...}` or `{"tag": "url", ...}` (or some other tagged format). But without any clear examples, I'm hesitant to code it. That is, if I guess wrong, and the spec is eventually clarified, are we going to have backward compatibility headaches ?

Are we making the same assumptions currently in the Android version ? If so, maybe I should code it.

Reply:
Indeed we do "support" aes success actions in the Android version though I don't know of any examples. However the specs does provide some encryption code and we used that in [our test unit](https://github.com/ACINQ/phoenix/blob/1dccf389c75e8cb7c0ef9136fb6a01538280845e/phoenix-legacy/src/androidTest/kotlin/fr/acinq/phoenix/LNUrlTest.kt#L40).

I think the correct solution would be to open an issue in the RFC repo and wait for test vectors.

Reply:
Issue created: https://github.com/fiatjaf/lnurl-rfc/issues/113